github-pr

```diff
@@ -12,11 +12,17 @@ def _check_required_fields(required, **args):
             sys.exit(1)
 
 
+def _return_status(commits):
+    for commit in commits:
+        for commit_status in commit.get_statuses():
+            return commit_status.state
+
 def _print_pr(pr, **args):
     if args['numberonly']:
         print "%d" % pr.number
     else:
-        print "#%d [%s] %10s <- %-30s    %s" % (pr.number, pr.state, pr.base.ref, pr.head.ref, pr.title)
+        pr_status = _return_status(pr.get_commits())
+        print "#%d [%s] %10s <- %-30s    %s - Merge Status:%s" % (pr.number, pr.state, pr.base.ref, pr.head.ref, pr.title, pr_status)
```

Comment:
I wouldn't have "Merge Status" in the output, it's just redundant on every line. Also, this has nothing to do with a merge status, it's just the status.